openedx/features/edly/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+
+class EdlyOrganization(models.Model):
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=100, unique=True, validators=[ALPHANUMERIC_VALIDATOR])
+
+    def __str__(self):
+        return u"{name} ({slug})".format(name=self.name, slug=self.slug)
+
+
+class EdlySubOrganization(models.Model):
+    edly_organization = models.ForeignKey(EdlyOrganization, null=False, blank=False)
+    edx_organization = models.OneToOneField(Organization)
+    lms_site = models.OneToOneField(Site, related_name='lms_site')
+    studio_site = models.OneToOneField(Site, related_name='studio_site')
+
+    def __unicode__(self):
```

Comment:
we only need `__str__` method https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods

Reply:
Done